schema/vrs.yaml

```diff
@@ -70,24 +79,30 @@ definitions:
 
   Allele:
     description: >-
-      The sequence state at a Location.
+      The state of a molecule at a :ref:`Location`.
```

Comment:
Interesting that you used the term `molecule` instead of `sequence` here. Wouldn't `sequence` be more concise?

Reply:
This change was made to align the YAML with the documentation. But `molecule` probably is a better anyways, as it also covers `ChromosomeLocation`.